pkg/scheduler/apis/config/v1alpha2/defaults.go

```diff
@@ -165,3 +165,25 @@ func SetDefaults_InterPodAffinityArgs(obj *v1alpha2.InterPodAffinityArgs) {
 		obj.HardPodAffinityWeight = pointer.Int32Ptr(1)
 	}
 }
+
+func SetDefaults_NodeResourcesLeastAllocatedArgs(obj *v1alpha2.NodeResourcesLeastAllocatedArgs) {
+	defaultResourceLeastAllocatedSet := []v1alpha2.ResourceSpec{
```

Comment:
put this in a common function that both `SetDefaults_NodeResourcesLeastAllocatedArgs` and `SetDefaults_NodeResourcesMostAllocatedArgs` call

Reply:
done